scripts/utils/cliCommandWrapper.spec.ts

```diff
@@ -39,7 +39,7 @@ describe('cliCommandWrapper', () => {
       expected: { key: 'value', flag: true, a: true, b: true, c: true },
     },
   ])('should parse arguments', async ({ inputArgs, schema, expected }) => {
-    process.argv = ['node', 'script.js', ...inputArgs]
+    process.argv = ['node', 'script.ts', ...inputArgs]
```

Comment:
@andrewi-wd I think our CLI command tests are actually broken, this shouldn't be passing, but it does.

Reply:
ah, we are not loading any actual file there, so looks like we are OK